Core/include/Acts/Seeding/GbtsGeometry.hpp

```diff
@@ -358,7 +360,7 @@ class GbtsGeometry {
 
   int m_nEtaBins{0};
 
-  std::unique_ptr<Acts::GbtsConnector> m_connector;
+  std::unique_ptr<Acts::Experimental::GbtsConnector> m_connector;
```

Comment:
```suggestion
  std::unique_ptr<GbtsConnector> m_connector;
```

should not be necessary after `namespace Acts::Experimental`